src/AppMenuSearch.cc

```
@@ -0,0 +1,434 @@
+{
+    if (!m_searchMenu) {
+        return;
+    }
+
+    m_lastSearchQuery = text;
+
+    // Clear results if search text is too short
+    if (isQueryTooShort(text)) {
+        clear();
+        resetSearchState();
+        Q_EMIT repositionRequested();
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)